frontend/index.html

```diff
@@ -1,57 +1,50 @@
 <!DOCTYPE html>
 <html lang="en">
-	<head>
-		<meta charset="UTF-8" />
-		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
-		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
-		<title>ETHSBell</title>
-		<link rel="stylesheet" href="index.css" />
-		<include src="include/prelude.html"></include>
-		<script defer src="/global/progress.js"></script>
-		<script defer src="index.js"></script>
-		<script src="redirect.js"></script>
-	</head>
 
-	<body class="background-color">
-		<include src="include/navbar.html"></include>
-		<div id="wrapper" class="background-color">
-			<div class="big block text-center content">
-				<template id="current_period_time_template">
-					<div class="progress">
-						<svg class="progress-ring">
-							<circle
-								class="progress-ring__circle"
-								stroke="currentColor"
-								fill="transparent"
-							/>
-							<circle
-								class="progress-ring__border"
-								stroke="currentColor"
-								fill="transparent"
-								strokewidth="4"
-							/>
-						</svg>
-					</div>
-					<span class="text-wrapper"
-						>{NAME} ends at <b>{END}</b> (in <b>{END_IN}</b>).
-					</span>
-				</template>
-				<div class="content">
-					<span id="schedulename"></span>
-					<div class="big" id="current_parent"></div>
-					<div id="next_parent">
-						<div class="divider"></div>
-						<template id="next_period_template">
-							The next period is {NAME}, which ends at {END}
-						</template>
-						<div class="bigish" id="next_period"></div>
-					</div>
-				</div>
+<head>
```

Comment:
We should figure out what's causing our editors to format html differently and configure it in the repo, it makes diffs difficult to read.

Reply:
Fixed